### DISCUSSION TITLE
How am I supposed to get the list of attributes?

### DISCUSSION THREAD

**Discussion by Nakilon** [rubycdp/ferrum#220] [Q&A]
I see some 2019 changelog that deprecates the `.attributes` but then what am I supposed to use? 
```none
> br.css(".outer_page > .inner_page").sample.property("attributes")
=> {"0"=>{}, "1"=>{}, "2"=>{}, "3"=>{}, "4"=>{}, "5"=>{}, "6"=>{}, "7"=>{}, "8"=>{}, "9"=>{}}
```
This is something wrong.

**Comment by Nakilon** [comment]
Did it like this for now:
```ruby
br.evaluate("Array.from(arguments[0].attributes).map(_=>_.name)", br.css(".outer_page > .inner_page").sample)
```

**Comment by route** [comment]
I think changelog doesn't show correct info, this is implemented in Cuprite rather than Ferrum and it was removed back then because it didn't work. So I think this is what we a re missing now in Ferrum.
